src/components/molecules/TableHeader/components/EditTableTitleModal/EditTableTitleModal.tsx

```diff
@@ -0,0 +1,170 @@
+  );
+
+  const {
+    register,
+    handleSubmit,
+    errors,
+    watch,
+    reset,
+  } = useForm<EditTableForm>({
+    mode: "onBlur",
+    reValidateMode: "onBlur",
+    defaultValues: formDefaultValues,
+  });
+
+  // Updates the default values whenever they change. This is required because they are cached after the first render.
+  useEffect(() => {
+    reset(formDefaultValues);
+  }, [formDefaultValues, reset, title]);
+
+  // use useAsyncFn for easier error handling, instead of state hook
+  const [{ error }, updateTables] = useAsyncFn(
+    async (values: EditTableForm) => {
+      if (!user || !tableOfUser || !venueId) return;
+
+      const updatedTable = {
+        ...tableOfUser,
+        title: values.title,
+        subtitle: values.subtitle,
+        capacity: values.capacity,
+      };
+
+      return await firebase.functions().httpsCallable("venue-updateTables")({
+        venueId,
+        tables,
+        updatedTable,
+      });
+    },
+    [tableOfUser, tables, user, venueId]
+  );
+
+  const values = watch();
+
+  const onSubmit = useCallback(async () => {
+    await updateTables(values).then(() => {
+      onHide();
+    });
+  }, [onHide, updateTables, values]);
+
+  return (
+    <Modal show={isShown} onHide={onHide}>
+      <Modal.Body>
+        <form onSubmit={handleSubmit(onSubmit)} className="EditTableTitleModal">
+          <InputField
+            ref={register({ required: true })}
+            name="title"
+            defaultValue={title}
+            containerClassName="EditTableTitleModal__input--spacing"
+            placeholder="Table topic"
+          />
+          {errors.title && errors.title.type === "required" && (
+            <span className="input-error">Topic is required</span>
+          )}
+
+          <InputField
+            ref={register}
+            name="subtitle"
+            containerClassName="EditTableTitleModal__input--spacing"
+            placeholder="Describe this table (optional)"
+          />
+
+          <div className="EditTableTitleModal__capacity">
+            <label className="EditTableTitleModal__max-capacity">
+              Number of seats (max {MAX_TABLE_CAPACITY})
+            </label>
+            <InputField
+              ref={register({
+                required: true,
+                max: MAX_TABLE_CAPACITY,
+                min: MIN_TABLE_CAPACITY,
+              })}
+              name="capacity"
+              type="number"
+              min={MIN_TABLE_CAPACITY}
+              max={MAX_TABLE_CAPACITY}
+              placeholder="Max seats"
+            />
+          </div>
+          {errors.capacity?.type === "required" && (
+            <span className="input-error">Capacity is required</span>
+          )}
+          {(errors.capacity?.type === "min" ||
+            errors.capacity?.type === "max") && (
+            <span className="input-error">
+              Capacity must be between {MIN_TABLE_CAPACITY} and{" "}
+              {MAX_TABLE_CAPACITY}
```

Comment:
```suggestion
              Capacity must be between {MIN_TABLE_CAPACITY} and
              {MAX_TABLE_CAPACITY}
```
My guess is that these things are put there by the linter

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/24da14326be4d256ec9efa8c3472f64899920adb

Reply:
But yes, it was the linter.

Reply:
@denisdimitrov @mike-lvov FYI, in future, this isn't something that needs changing. The code is clearer/simpler leaving it as it was. I would consider the change made in https://github.com/sparkletown/sparkle/commit/24da14326be4d256ec9efa8c3472f64899920adb for this to be a regression, and personally would revert it.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/f6e11778a3edad587868e67cdb5bb100ff16473d